lib/cmd/fhc/target.js

```diff
@@ -104,32 +97,6 @@ function target(argv, cb) {
         ini.set(keys.KEY_ID, targ[keys.KEY_ID], "user");
       }
 
-      var _gotVersion = function() {
-        ini.save(function(err) {
-          var message;
-          if (targ.user) {
-            message = " User: " + targ.user;
-          } else {
-            message = " API Key: " + targ[keys.KEY_ID];
-          }
-          target.message = i18n._("Successfully targeted: ") + targ.target + message;
-          return cb(err, targ);
-        });
-      };
-
-      ngui({_ : []}, function(err, isFh3) {
-        if (err) {
-          if (targ.fhversion) {
-            ini.set('fhversion', targ.fhversion);
-          } else {
-            target.message += i18n._('Your login has expired - please log in again!\n ') + login.usage;
-          }
-          return _gotVersion();
-        }
-        var version = (isFh3) ? 3 : 2;
-        ini.set('fhversion', version);
-        return _gotVersion();
-      });
```

Comment:
Removal of these lines causes `fhc ERR! cb() never called!`  You have to call `ini.save(...` function that calls the `cb` callback. Note that useful message `Successfully targeted:...` is composed there as well